src/test/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/CreditorsAfterOneYearServiceImplTests.java

```diff
@@ -0,0 +1,191 @@
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorsafteroneyear.Total;
+import uk.gov.companieshouse.web.accounts.service.smallfull.BalanceSheetService;
+import uk.gov.companieshouse.web.accounts.service.smallfull.CreditorsAfterOneYearService;
+import uk.gov.companieshouse.web.accounts.transformer.smallfull.CreditorsAfterOneYearTransformer;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CreditorsAfterOneYearServiceImplTests {
+
+    @InjectMocks
+    private CreditorsAfterOneYearService mockCreditorsAfterOneYearService = new CreditorsAfterOneYearServiceImpl();
+
+    @Mock
+    CreditorsAfterOneYearTransformer mockTransformer;
+
+    @Mock
+    private CreditorsAfterOneYearResourceHandler mockCreditorsAfterOneYearResourceHandler;
+
+    @Mock
+    BalanceSheetService mockBalanceSheetService;
+
+    @Mock
+    BalanceSheet mockBalanceSheet;
+
+    @Mock
+    private CreditorsAfterOneYearGet mockCreditorsAfterOneYearGet;
+
+    @Mock
+    private ApiClientService mockApiClientService;
+
+    @Mock
+    private ApiClient mockApiClient;
+
+    @Mock
+    private SmallFullResourceHandler mockSmallFullResourceHandler;
+
+    private static final String TRANSACTION_ID = "transactionId";
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+
+    private static final String COMPANY_NUMBER = "companyNumber";
+
+    private static final String BASE_SMALL_FULL_URI = "/transactions/" + TRANSACTION_ID +
+            "/company-accounts/" + COMPANY_ACCOUNTS_ID +
+            "/small-full";
+
+    private static final String CREDITORS_AFTER_ONE_YEAR_URI = BASE_SMALL_FULL_URI + "/notes/creditors-after-more-than-one-year";
+
+
+    @Test
+    @DisplayName("GET - Creditors After One Year successful path")
+    void getCreditorsAfterOneYearSuccess() throws Exception {
+
+        CreditorsAfterOneYearApi creditorsAfterOneYearApi = new CreditorsAfterOneYearApi();
+        getMockCreditorsAfterOneYearApi(creditorsAfterOneYearApi);
+
+        when(mockTransformer.getCreditorsAfterOneYear(creditorsAfterOneYearApi)).thenReturn(createCreditorsAfterOneYear());
+        when(mockBalanceSheetService.getBalanceSheet(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, COMPANY_NUMBER)).thenReturn(mockBalanceSheet);
+        when(mockBalanceSheet.getBalanceSheetHeadings()).thenReturn(new BalanceSheetHeadings());
+
+        CreditorsAfterOneYear creditorsAfterOneYear = mockCreditorsAfterOneYearService.getCreditorsAfterOneYear(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, COMPANY_NUMBER);
+
+        assertNotNull(creditorsAfterOneYear);
+        assertNotNull(creditorsAfterOneYear.getOtherCreditors());
+        assertNotNull(creditorsAfterOneYear.getOtherCreditors().getCurrentOtherCreditors());
+        assertNotNull(creditorsAfterOneYear.getTotal());
+        assertNotNull(creditorsAfterOneYear.getTotal().getCurrentTotal());
```

Comment:
We could move all these asserts to a separate method e.g. `validateCreditorsAfterOneYear`so that you could reuse it in the `getCreditorsAfterOneYearSuccessHttpStatusNotFound()` test